__tests__/app.test.js

```diff
@@ -66,14 +66,62 @@ describe("app", () => {
         });
     });
   });
-  test("400: invalid sort by query", () => {
-    return request(app)
-      .get("/api/reviews?sort_by=invalid_sort")
-      .expect(400)
-      .then(({ body }) => {
-        expect(body.msg).toBe("Bad Request");
-      });
+  describe("/api/reviews?sort_by", () => {
+    test("400: invalid sort by query", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=invalid_sort")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Invalid Request");
+        });
+    });
   });
-});
-
-//test for 404 bad path misspelled
+  describe("/api/review/:review_id", () => {
+    test("200: GET responds with a single review object", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual({
```

Comment:
In order to make this test more flexible I would suggest using a different matcher (such as objectContaining or toMatchObject) to allow the test to still pass if later on we add more collumns. This prevents us having to go back and change the test